packages/react-button/src/components/CommandBarButton/CommandBarButton.scss

```diff
@@ -0,0 +1,48 @@
+@import '~@fluentui/common-styles/dist/sass/themeCssVariables';
+@import '~@fluentui/common-styles/dist/sass/highContrast';
+@import '~@fluentui/common-styles/dist/sass/focusBorder';
+
+.root {
```

Comment:
pls take a look at Mak's PR https://github.com/microsoft/fluentui/pull/14855
you are missing few things:
1) we should move to `useClasses` instead of raw scss.
2) let's add screener tests for this new button